Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'http://rubygems.org'
 
-ruby '2.1.1'
+ruby '2.1.7'
```

Comment:
Why did you decide to bump the Ruby version in this commit? Was it required for other things? Since I don't think we're requiring Docker for development, you should also bump the version in the `.ruby-version` which is used by rvm / rbenv.